blogposts/2020/announcing-sourcegraph-3.17.md

```diff
@@ -0,0 +1,258 @@
+---
+title: "Sourcegraph 3.17: Faster and automatic precise code intelligence, preview of code insights, and AND/OR queries for searching file contents"
+author: Adam Herzog
+publishDate: 2020-06-20T10:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-3.17
+heroImage: /blog/3.17-release-blog-img.jpg
+published: true
+---
+
+Explore, navigate, and better understand all code, everywhere, faster with Sourcegraph Universal Code Search. Uber, Yelp, Nutanix, Tinder, and other enterprises rely on Sourcegraph to improve developer productivity and manage large-scale code changes.
+
+<div style="padding-left: 2rem">
+
+[**🧠 Faster and automatic precise code intelligence**](#faster-and-automatic-precise-code-intelligence)<br />
+Precise code intelligence is now 35% faster than in 3.16. Automatic code intelligence (🧪 experimental feature) will bring precise results to more repositories faster.
+
+[**📊 Product preview: Code insights**](#product-preview-code-insights)<br />
+Help developers be more productive and do their jobs better by aggregating data in Sourcegraph.
+
+[**💻 Sourcegraph developer features**](#sourcegraph-developer-features)<br />
+Introducing AND/OR queries for searching file contents, updating the Gitolite exclude pattern, and improving debugging.
+
+[**🛠 Sourcegraph admin features**](#sourcegraph-admin-features)<br />
+Easier alerting configuration, repository permission syncing on by default, and notifications when Sourcegraph is out-of-date.
+
+[**📝 Changelog**](#changelog)<br />
+Every detail that changed in this release
+
+[**🎖️ Thank you**](#thank-you)<br />
+Sourcegraph couldn't be what it is without the community.
+
+</div>
+
+**Deploy or upgrade:** [Local](https://docs.sourcegraph.com/#quickstart-guide) | [AWS](https://github.com/sourcegraph/deploy-sourcegraph-aws) | [DigitalOcean](https://marketplace.digitalocean.com/apps/sourcegraph?action=deploy&refcode=48dfb3ccb51c) | [Kubernetes cluster](https://github.com/sourcegraph/deploy-sourcegraph)
+
+<div class="alert alert-info">
+  <p>
+    <strong>LEARN MORE:</strong><br />
+    <a href="https://info.sourcegraph.com/online-meetup-june25?utm_source=blog">
+      Online meetup: Sourcegraph 3.17 - Thursday, June 21 at 10AM PDT
+    </a>
+  </p>
+  </div>
+
+---
+
+## Faster and automatic precise code intelligence
```

Comment:
There were a few edits that no longer match the shared text in one of the blog posts. I've started to match them up here - could we make sure that the text isn't too different before submission? Feel free to make edits in this PR and merge along with the announcement:

https://github.com/sourcegraph/about/pull/1086

Reply:
it's okay if the text in the blog posts don't match 100% - do you have an example of what you're referring to?

Reply:
I think I resolved this. See incoming changes.